src/browser/modules/D3Visualization/components/DetailsPane.tsx

```diff
@@ -56,103 +56,66 @@ export function DetailsPaneComponent({
   return (
     <>
       <PaneHeader>
+        <PaneTitle>
+          {upperFirst(vizItem.type)} Properties{' '}
+          <ClipboardCopier
+            textToCopy={allItemProperties
+              .map(prop => `${prop.key}: ${prop.value}`)
+              .join('\n')}
+            titleText="Copy all properties to clipboard"
+            iconSize={10}
+          />
+        </PaneTitle>
         {vizItem.type === 'relationship' && (
-          <StyledTokenRelationshipType
-            style={{
-              backgroundColor: graphStyle
-                .forRelationship(vizItem.item)
-                .get('color'),
-              color: graphStyle
-                .forRelationship(vizItem.item)
-                .get('text-color-internal'),
-              cursor: 'default'
+          <StylableRelType
```

Comment:
Rather than repeat the popup/grass editor pattern I've broken it out to a component